docs/readers.md

```diff
@@ -37,7 +37,7 @@ with COGReader("myfile.tif"s) as cog:
     assert img.data.count == 1
 ```
 
-- **part()**: Read part of a raster
+- **part()**: Read a raster for a given bounding box (`bbox`)
```

Comment:
Might be good to note that this is in image coordinates, not geospatial coordinates.

Reply:
> Might be good to note that this is in image coordinates, not geospatial coordinates.

well that's the opposite, `part()` takes a WGS84 bounding box, but yes 👍